src/main/java/goodgenerator/blocks/tileEntity/GTMetaTileEntity/YOTTAHatch.java

```diff
@@ -224,23 +225,27 @@ public void onFirstTick(IGregTechTileEntity aBaseMetaTileEntity) {
 
     @Override
     public void onPostTick(IGregTechTileEntity aBaseMetaTileEntity, long aTick) {
-        IGridNode node = getGridNode(null);
-        if (node != null) {
-            IGrid grid = node.getGrid();
-            if (grid != null) {
-                grid.postEvent(new MENetworkCellArrayUpdate());
-                IStorageGrid storageGrid = grid.getCache(IStorageGrid.class);
-                if (storageGrid == null) {
-                    node.getGrid().postEvent(new MENetworkStorageEvent(null, StorageChannel.FLUIDS));
-                } else {
-                    node.getGrid()
-                            .postEvent(
-                                    new MENetworkStorageEvent(storageGrid.getFluidInventory(), StorageChannel.FLUIDS));
+        ticksSinceUpdate++;
+        if (ticksSinceUpdate > 20) {
+            IGridNode node = getGridNode(null);
+            if (node != null) {
+                IGrid grid = node.getGrid();
+                if (grid != null) {
+                    grid.postEvent(new MENetworkCellArrayUpdate());
+                    IStorageGrid storageGrid = grid.getCache(IStorageGrid.class);
+                    if (storageGrid == null) {
+                        node.getGrid().postEvent(new MENetworkStorageEvent(null, StorageChannel.FLUIDS));
+                    } else {
+                        node.getGrid()
+                                .postEvent(new MENetworkStorageEvent(
+                                        storageGrid.getFluidInventory(), StorageChannel.FLUIDS));
+                    }
+                    node.getGrid().postEvent(new MENetworkCellArrayUpdate());
                 }
-                node.getGrid().postEvent(new MENetworkCellArrayUpdate());
             }
+            super.onPostTick(aBaseMetaTileEntity, aTick);
```

Comment:
the `super.onPostTick(aBaseMetaTileEntity, aTick);` should be passed every tick so put it outside the brackets